Changelog.md

```diff
@@ -1,3 +1,9 @@
+## 4.4.2 - unreleased
+
+### Fixed
+
+-   Now DECT modem firmware is correctly detected for 91x family.
```

Comment:
```suggestion
-   The DECT modem firmware is now correctly detected for nRF91x1 devices.
```
91x is not allowed, and 91x is not a family: https://nordicsemi.atlassian.net/wiki/spaces/MAR/pages/132940950/Product+names